src/Clapi/PathQ.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE QuasiQuotes, TemplateHaskell #-}
+module Clapi.PathQ (pathq) where
+import Language.Haskell.TH.Quote
+import Language.Haskell.TH
+import Data.Maybe (fromJust)
+
+import Path.Parsing (fromString)
+
+qe :: String -> Q Exp
+qe ps = fromString ps >>= const [| fromJust $ fromString ps |]
```

Comment:
```haskell
{-# LANGUAGE StandaloneDeriving, DeriveLift #-}

import Language.Haskell.TH.Lift  -- need th-lift in your cabal file
...
deriving instance Lift Path

qe :: String -> Q Exp
qe ps = fromString ps >>= lift
```

That definition is short enough to inline ;-)

Reply:
I can't get this to work because it can't derive lift for Text.

Reply:
Add `th-lift-instances` to your cabal file too, and I think it will go.